code/game/machinery/doors/blast_door.dm

```diff
@@ -146,6 +147,10 @@
 /obj/machinery/door/blast/close()
 	if (src.operating || (stat & BROKEN || stat & NOPOWER))
 		return
+	if(shutter)
```

Comment:
With the above noted, it could just become
`crush(shutter_damage)` with no ifelse statement.